server/pull_request.go

```diff
@@ -75,6 +81,9 @@ func (s *Server) handlePullRequestEvent(event *PullRequestEvent) {
 				s.handleUpdateSpinWick(pr, false)
 			}
 		}
+		if s.isBlockPRMergeInLabels(pr.Labels) {
```

Comment:
Why do we need to check for this here? Isn't checking on label switches enough?

Reply:
yes, but this is just to update the check to the latest commit hash. i can remove @hanzei

Reply:
Makes sense :+1: